srv/src/main/java/my/bookshop/handlers/AdminServiceAuditHandler.java

```diff
@@ -50,52 +44,46 @@
 		this.auditLog = auditLog;
 	}
 
-	@After(event = { CqnService.EVENT_READ })
-	public void afterReadOrder(Stream<Orders> orders) {
-		orders.forEach(order -> {
-			if (order.getOrderNo() != null) {
-				this.auditLog.logDataAccess(createAccess(order));
-			}
-		});
-	}
-
 	@Before(event = { CqnService.EVENT_CREATE })
 	public void beforeCreateOrder(Stream<Orders> orders) {
 		orders.forEach(order -> {
-			DataModification dataModification = createDataModification(order, null, Action.CREATE);
-			if (dataModification != null) {
-				this.auditLog.logDataModification(Arrays.asList(dataModification));
+			ConfigChange cfgChange = createConfigChange(order, null);
+			if (cfgChange != null) {
```

Comment:
`cfgChange `will nevver be null, won't it?

Reply:
right, will never be null, will remove check